src/Conversion/ONNXToKrnl/Tensor/Resize.cpp

```diff
@@ -129,12 +129,27 @@ struct ONNXResizeOpLowering : public ConversionPattern {
 
     // Call external function when the mode is not "nearest"
     // Create KrnlCallOp and replace the du chain
+    // One of inputs, scales() and size(), has to be None.
```

Comment:
One of the inputs, 'scales' or 'size', has to be 'None'.

Reply:
Yes, one of them has to be None. In general, the semantics of None is interpreted at the ONNX Op level. We could use tensor<0xT> for None and lower it to memref. So that we can keep None in Krnl and llvm after ONNX Op. We may have another PR for that functionality.